src/test/java/ProjectTest.java

```diff
@@ -0,0 +1,208 @@
+        String testUsername = "admin";
+
+        UserWithProjectResponse response = litmusClient.getUserWithProject(testUsername);
+
+        assertThat(response.getUsername()).isEqualTo(testUsername);
+        assertThat(response.getProjects()).isNotNull();
+        assertThat(response.getProjects().size()).isGreaterThanOrEqualTo(1);
+
+        ProjectResponse firstProject = response.getProjects().get(0);
+        assertThat(firstProject.getProjectID()).isNotNull();
+        assertThat(firstProject.getName()).isNotNull();
+    }
+
+
+    @Test
+    public void testGetProjectsStats() throws IOException, LitmusApiException {
+        List<ProjectsStatsResponse> response = litmusClient.getProjectsStats();
+
+        assertThat(response.size()).isGreaterThanOrEqualTo(1);
+
+        ProjectsStatsResponse firstProject = response.get(0);
+        assertThat(firstProject.getProjectID()).isNotNull();
+        assertThat(firstProject.getName()).isNotNull();
+        assertThat(firstProject.getMembers().getTotal()).isGreaterThanOrEqualTo(1);
+
+        ProjectsStatsResponse.Members members = firstProject.getMembers();
+        assertThat(members.getOwner().size()).isGreaterThanOrEqualTo(1);
+
+        ProjectsStatsResponse.Owner firstOwner = members.getOwner().get(0);
+        assertThat(firstOwner.getUserID()).isNotNull();
+        assertThat(firstOwner.getUsername()).isNotNull();
+    }
+
+    @Test
+    public void testGetProjectMembers() throws IOException, LitmusApiException {
+        String projectID = "63488419-fc5c-4a1a-8aa9-4e736e8b3a73";
+        String status = "active";
+
+        List<ProjectMemberResponse> response = litmusClient.getProjectMembers(projectID, status);
+
+        assertThat(response.size()).isGreaterThanOrEqualTo(1);
+
+        ProjectMemberResponse firstMember = response.get(0);
+        assertThat(firstMember.getUserID()).isNotNull();
+        assertThat(firstMember.getUsername()).isNotNull();
+
+        if (firstMember.getDeactivatedAt() != null) {
+            assertThat(firstMember.getDeactivatedAt()).isGreaterThan(0L);
+        }
+    }
+
+    @Test
+    public void testGetProjectOwners() throws IOException, LitmusApiException {
+        String projectID = "63488419-fc5c-4a1a-8aa9-4e736e8b3a73";
+
+        List<ProjectMemberResponse> response = litmusClient.getProjectOwners(projectID);
+
+        assertThat(response.size()).isGreaterThan(0);
+        ProjectMemberResponse firstOwner = response.get(0);
+        assertThat(firstOwner.getUserID()).isNotNull();
+        assertThat(firstOwner.getUsername()).isNotNull();
+        assertThat(firstOwner.getRole()).isEqualTo("Owner");
+
+    }
+
+    @Test
+    public void testDeleteProject() throws IOException, LitmusApiException {
+        String projectID = "63488419-fc5c-4a1a-8aa9-4e736e8b3a73";
+
+        CommonResponse response = litmusClient.deleteProject(projectID);
+
+        assertThat(response.getMessage()).isEqualTo("Successfully deleted project.");
+    }
```

Comment:
@somln, We need to test our entire test code repeatedly. I think we can test this just once because we remove project that has projectID "63488419-fc5c-4a1a-8aa9-4e736e8b3a73". And next entire test, maybe some tests break because project "63488419-fc5c-4a1a-8aa9-4e736e8b3a73" is not existed on litmus project.

Reply:
I think use this BDD pattern on entire unit test.